etl/interfaces/interfaces.go

```diff
@@ -107,6 +108,8 @@ type StorageSetter interface {
 type StorageAPIFetcher interface {
 	// GetRecord returns record with provided reference from db.
 	GetRecord(ref models.Reference) (models.Record, error)
```

Comment:
Do we need `GetRecord` now? Or `GetState` completely replaced it?

Reply:
I think yes, because this is used for tests until we completely delete the record table